vllm/v1/kv_offload/abstract.py

```diff
@@ -67,6 +71,17 @@ class OffloadingEvent:
 
 
 class OffloadingManager(ABC):
+    @abstractmethod
+    def __init__(self, backend: "Backend", enable_events: bool = False):
```

Comment:
This adds a dependency between the `OffloadingManager` and `Backend`.
Not all OffloadingManagers would want this abstraction.
Thus I think we should remove this.